packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/index.jsx

```diff
@@ -25,6 +25,15 @@ import Contracts from './Contracts'
 
 import { useI18n } from 'cozy-ui/transpiled/react'
 
+const contractRx = /io\.cozy\.[a-z]+\.accounts/
```

Comment:
Is it specified somewhere? If I read https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.bank.md#iocozybankaccounts it doesn't say anything related to "contracts" just about bank account, which is for me, not the same. 

Do we have a paper discussing about a potential io.cozy.contracts doctype or something similar? Personally, I don't really like this `*.accounts` magic stuff.

Reply:
Also this stuff should be in cozy-client IMHO